hail/python/test/hailtop/utils/test_utils.py

```diff
@@ -1,7 +1,11 @@
 from hailtop.utils import (partition, url_basename, url_join, url_scheme,
                            url_and_params, parse_docker_image_reference, grouped)
+<<<<<<< HEAD
 from hailtop.utils.utils import unzip
+=======
```

Comment:
You'll need to fix this to combine the two imports on one line and remove the lines that start with `====` and `<<<` and `>>>`.

```
from hailtop.utils.utils import unzip, digits_needed
```

Reply:
Fixed